app-emulation/cap32_libretro/cap32_libretro-4.2.0_20171109.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A port of Caprice32, an Amstrad CPC emulator, to the libretro API"
+DESCRIPTION="Caprice32 is a software emulator of the Amstrad CPC 8bit home \
+computer series. The emulator faithfully imitates the CPC464, CPC664, and \
+CPC6128 models."
+HOMEPAGE="http://sourceforge.net/projects/caprice32/"
```

Comment:
```diff
-HOMEPAGE="http://sourceforge.net/projects/caprice32/"
+HOMEPAGE="https://sourceforge.net/projects/caprice32/
+	https://github.com/libretro/libretro-cap32"
-COPYRIGHT="1997-2018, Ulrich Doewich, dantoine the libretro team"
+COPYRIGHT="1997-2018 Ulrich Doewich, dantoine the libretro team"
```
Yep, it is possible to have more than one `HOMEPAGE`.
I don't know, however, if that's a good idea, for this recipe.
Anyway, if you keep just the SF homepage, there is a small change: `s/http/https/`

Reply:
Hmm sure why not but we've never done this before for the other cores?
Pushed recipe with "https"